libraries/ms-drivers/inc/adt_7476a_fan_controller.h

```diff
@@ -0,0 +1,38 @@
+#pragma once
+
+// Requires interrupts, soft timers, GPIO, and GPIO interrupts to be initialized
+
+typedef void (*Adt7476ADataCallback)(void *context);
```

Comment:
minor nitpick: This should be `Adt7476aDataCallback` rather than `Adt7476ADataCallback` to match the convention you set with the storage and settings type names